frontend/src/pages/docs/[...slug].tsx

```diff
@@ -8,6 +8,8 @@ import {
   MDXRemoteProps,
   MDXRemoteSerializeResult,
 } from "next-mdx-remote";
+import rehypeMathJax from 'rehype-mathjax';
+import remarkMath from "remark-math";
```

Comment:
Shouldn't there be a package.json diff?

Reply:
Definitely added it to the commit, not sure why it's not there...trying again